config/app.php

```diff
@@ -149,6 +149,7 @@
         Illuminate\Redis\RedisServiceProvider::class,
         Illuminate\Auth\Passwords\PasswordResetServiceProvider::class,
         Illuminate\Session\SessionServiceProvider::class,
+        // Laravel\Tinker\TinkerServiceProvider::class,
```

Comment:
Good call. Tinker was moved to a package in Laravel 5.4. So I guess our options are to either remove this commented out line, or install the tinker package separately. I'm leaning towards just removing this line. Thoughts?